js/src/Ice/ObjectAdapter.js

```diff
@@ -56,11 +56,13 @@ const _suffixes = [
     "Router.LocatorCacheTimeout",
     "Router.InvocationTimeout",
     "ProxyOptions",
+    "ThreadPool.Serialize",
```

Comment:
These properties have no effect in JS, since there is no thread pool.

We should consider removing them. We could also update PropertyNames.xml to mark some properties as "java only" or "for all except js".